### PR TITLE
Switch back to ondrej's mysql PPA

### PR DESCRIPTION
**Pull request by jibsheet**
Longer reasoning in the comment in the file

@edx/devops

FYI @nedbat

### CODE REVIEW

**Comment by nedbat** [comment]
@jibsheet so this should be correct for xenial also?

**Comment by jibsheet** [comment]
@nedbat correct.  You mentioned you had picked it to your branch, but I don't think you pushed (I got errors on your branch until I brought this in manually).

File: docker/plays/mysql.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- name: Deploy autom
```
Comment (e0d) [inline docker/plays/mysql.yml]: nit: spelling

**Review by e0d — APPROVED** [review]
one nit, but I approve.

**Comment by nedbat** [comment]
@jibsheet oops, right, I didn't push the cherry-pick.

**Pull request merged**